docs/about/overview/mesh-alg.mdx

```diff
@@ -1,23 +1,24 @@
 ---
 id: mesh-alg
```

Comment:
My only comment on this file is in the future we may want to make this document more accessible to newcomers to the project.